<launch>
	<node
		pkg = "jaguar4x4"
		type = "keyboard_teleop"
		name = "keyboard_teleop"
		required = "true"
		launch-prefix = "xterm -e"
	/>
	<node
		pkg="jaguar4x4"
		type="jaguar4x4_node"
		name="jaguar_node"
		respawn = "true"
		output = "screen"
	/>
</launch>